util/year.py

```diff
@@ -5,6 +5,8 @@
 import model
 from db import session
 
+fake_year_shim_enabled = False
```

Comment:
Osobně bych tady nechal jen jednu proměnnou (konkrétní id) a pak se ptal, jestli je definována (tedy poslední ročník = zakomentovat řádek s nastavením proměnné). Ale to je jen detail.